packages/shared/src/utils/helpers/barrier.ts

```diff
@@ -61,6 +61,6 @@ export const getAccumulatorBarriers = (
 };
 
 export const getBarrierPipSize = (barrier: string) => {
```

Comment:
`getBarrierPipSize` is used only in _packages/trader/src/Stores/Modules/Trading/trade-store.js_ for calculating `barrier_pipsize` based on value of current `barrier_1`. As `barrier_1` is string (+ it's specified in TS above) I removed extra type coercion ( line 65 `toString()` ).
According to card's requirement, we don't need `Math.floor(+barrier) === +barrier` and `+barrier % 1 === 0 `anymore, as they change step of incrementing/ decrementing barrier value: `getBarrierPipSize('4.00')` will return 0 (numbers after decimal point) and we get 3.99 -> 4 -> 5 and etc.